test_wsgi_kerberos.py

```diff
@@ -31,6 +31,7 @@ def test_authentication_missing_but_not_required(self, clean, name, response, st
         self.assertEqual(r.body, b'Hello ANONYMOUS')
         self.assertEqual(r.headers.get('WWW-Authenticate'), None)
         self.assertEqual(r.headers['content-type'], 'text/plain')
+        self.assertEqual(r.headers['content-length'], '15')
```

Comment:
I'm not sure that it makes sense to test for content-length on every single test. It makes sense to test that on 401/403 responses because that's in the wsgi_kerberos code, but for 200 responses, you're basically testing the webtest library, not wsgi_kerberos.